target/linux/generic/patches-4.4/150-device_tree_cmdline.patch

```diff
@@ -0,0 +1,233 @@
+--- a/arch/mips/boot/compressed/decompress.c
```

Comment:
Please keep the patches intact (with full commit log etc). also you need to update brcm63xx (and any other non-upstream users if there are any) as well, else you break them.